go.mod

```diff
@@ -94,9 +95,9 @@ require (
 	k8s.io/component-base v0.23.9 // indirect
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9 // indirect
 	open-cluster-management.io/api v0.6.1-0.20220208144021-3297cac74dc5 // indirect
+	open-cluster-management.io/multicloud-operators-subscription v0.10.0
```

Comment:
I think this needs a `go mod tidy`, because I don't think it's actually used.

Reply:
Gus added... but after removing that line then `go mod tidy`, looks fine